src/Fixer/Basic/BracesFixer.php

```diff
@@ -478,7 +482,13 @@ function ($item) {
 
                 if (!$isAnonymousClass && $prevToken->isWhitespace() && false !== strpos($prevToken->getContent(), "\n")) {
                     if (!$tokens[$startBraceIndex - 2]->isComment()) {
-                        $tokens->ensureWhitespaceAtIndex($startBraceIndex - 1, 1, ' ');
+                        $tokens->ensureWhitespaceAtIndex(
+                            $startBraceIndex - 1,
+                            1,
+                            self::LINE_NEXT === $this->configuration['position_after_return_type_hint'] && $token->isGivenKind(T_FUNCTION) && ':' === $tokens[$tokens->getNextNonWhitespace($closingParenthesisIndex)]->getContent() ?
```

Comment:
then you encountered custom token.
PHP's native `token_name` is not aware about custom tokens.

We do transform of some tokens to different them here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/tree/master/src/Tokenizer/Transformer
like what is `class` token? is it about defining new class  (`class Foo {}`) or getting it's name (`Foo::class`) ? you don't know, that's why we introduced new, custom token for later.

Use `$token->getName()` to get name of token (regardless it's custom or native), or `$token->toArray()`/`$token->toJson()` to dump the token properties